src/Synapse/Synapse/Common/AzureSessionCredential.cs

```diff
@@ -12,16 +12,16 @@
 // limitations under the License.
```

Comment:
Do the affected modules needed to have a release with the change? If yes, could you please update the changelog for it?